docs/source/user_guide/computed_properties.rst

```diff
@@ -130,12 +133,22 @@ Output files
       </tbody>
     </table>
 
-Example: Visualization of a Morphology
-"""""""""""""""""""""""""""""""""""""""""
 
-You can visualize morphologies using common molecule visualization tools such as `jmol <https://jmol.sourceforge.net/>`_ or `pymol <https://www.pymol.org/>`_. 
+.. ToDo: move to some extra section. For instance, third-party software or visualizing outputs or examples. This section is about computed properties. It is big and complex enough without this.
+
+Example of ``structure.cml`` and ``structure.mol2`` files (visualization)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+You can visualize the content of **cml** and **mol2** files using common molecule visualization tools such as `jmol <https://jmol.sourceforge.net/>`_ or `pymol <https://www.pymol.org/>`_.
+
+**Visualisation of** ``structure.cml`` **with jmol**:
+
+    .. image:: computed_properties/morph_viz_jmol.png
```

Comment:
I prefer to have first instructions and then the result, but I don't insist. Just make sure that the last step of the instructions is consistent (figure below / figure above)